src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -20,21 +21,10 @@ <h1>{{ title }}</h1>
   Could not verify email address. The verification link has expired.
   {% endtrans %}
 {% elif state == "verification_pending" %}
-  {% trans %}
-  Working... Please be patient.
-  {% endtrans %}
   <div class="loader"></div>
-{% elif state == "verification_failed" %}
-  {% trans %}
-  Failed to send confirmation email. Please try again.
-  If you no longer have access to your email address, please update your mozilla account email address.
-  <a
-    href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts"
-    target="_blank"
-  >
-    Change email address
-  </a>
-  {% endtrans %}
+    {% trans %}
+    Working... Please be patient.
+    {% endtrans %}
 {% elif state == "verification_timedout" %}
   {% trans %}
   This is taking longer than expected. Try again.
```

Comment:
Can we clarify what "This" is and what we want users to "Try again"?

Reply:
Updated